ui/app/[locale]/(authorized)/kafka/[kafkaId]/nodes/rebalances/RebalanceTable.tsx

```diff
@@ -0,0 +1,294 @@
+              isDisabled: row.meta?.allowedActions.includes("approve"),
+            },
+            {
+              title: t("refresh"),
+              onClick: () => onRefresh(row),
+            },
+            {
+              title: t("stop"),
+              onClick: () => onStop(row),
+              isDisabled: !row.meta?.allowedActions.includes("approve"),
+            },
+          ]}
+        />
+      )}
+      getExpandedRow={({ row }) => {
+        return (
+          <Flex className={"pf-v5-u-pl-xl"}>
+            <DescriptionList columnModifier={{ lg: "3Col" }}>
+              <FlexItem>
+                <DescriptionListGroup>
+                  <DescriptionListTerm>{t("mode")}</DescriptionListTerm>
+                  <DescriptionListDescription>
+                    {row.attributes.mode}
+                  </DescriptionListDescription>
+                </DescriptionListGroup>
+              </FlexItem>
+              <FlexItem className={"pf-v5-u-ml-auto-on-xl"}>
+                <DescriptionListGroup>
+                  <DescriptionListTerm>
+                    {t("auto_approval_enabled")}
+                  </DescriptionListTerm>
+                  <DescriptionListDescription>
+                    {row.meta.autoApproval === true ? "true" : "false"}
+                  </DescriptionListDescription>
+                </DescriptionListGroup>
+              </FlexItem>
+              <FlexItem className={"pf-v5-u-ml-auto-on-xl"}>
+                <DescriptionListGroup>
+                  <DescriptionListTerm>{t("brokers")}</DescriptionListTerm>
+                  <DescriptionListDescription>
+                    {row.attributes.brokers?.join(",")}
+                  </DescriptionListDescription>
+                </DescriptionListGroup>
+              </FlexItem>
+            </DescriptionList>
+          </Flex>
+        );
+      }}
+      filters={{
+        Name: {
+          type: "search",
+          chips: filterName ? [filterName] : [],
+          onSearch: onFilterNameChange,
+          onRemoveChip: () => {
+            onFilterNameChange(undefined);
+          },
+          onRemoveGroup: () => {
+            onFilterNameChange(undefined);
+          },
+          validate: (value) => true,
+          errorMessage: "At least 3 characters",
+        },
+        Status: {
+          type: "checkbox",
+          chips: filterStatus || [],
+          onToggle: (status) => {
+            const newStatus = filterStatus?.includes(status)
+              ? filterStatus.filter((s) => s !== status)
+              : [...filterStatus!, status];
+            onFilterStatusChange(newStatus);
+          },
+          onRemoveChip: (status) => {
+            const newStatus = (filterStatus || []).filter((s) => s !== status);
+            onFilterStatusChange(newStatus);
+          },
+          onRemoveGroup: () => {
+            onFilterStatusChange(undefined);
+          },
+          options: StatusLabel,
+        },
+        mode: {
```

Comment:
```suggestion
        Mode: {
```

Also, can mode function like the status search with a fixed list of values that can be chosen with select boxes? There are only three values: `full`, `add-brokers`, and `remove-brokers`.

Reply:
updated